packages/client/src/Conversation.test.tsx

```diff
@@ -29,3 +30,18 @@ it("displays a loading indicator while the conversation is leading", () => {
   )
   expect(app).toIncludeText("Loading...")
 })
+
+it("collapses read messages in a conversation", () => {
```

Comment:
In its present form this test does not make any assertions about whether a card is collapsed. A good practice to follow is the "red-green" cycle: write a test first, make sure that it fails, write code until the test passes. Writing the test first is ideal; but the next best thing is to temporarily disable the feature you worked on and make sure that the test fails. I noticed that this test passes even if `expanded` is always set to `true` or to `false`.

What would be great is a test that explicitly checks that a read presentable is collapsed, and that an unread presentable is not collapsed.